### PR TITLE
Issue18 launch game

### PR DESCRIPTION
**Pull request by rupsis**
User can now run a game from the game menu

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -67,6 +67,9 @@ def start_menu ():
           keepGoing = False
           break
 
+        elif event.key == pygame.K_SLASH:
```
Comment (DannyPeck) [inline main.py]: Why K_SLASH?  Just curious :).

File: main.py [hunk 2]
```diff
@@ -10,8 +10,8 @@
 
 def start_menu ():
   window_size = (WINDOW_WIDTH, WINDOW_HEIGHT)
-  # screen = pygame.display.set_mode ((0, 0), pygame.FULLSCREEN)
-  screen = pygame.display.set_mode (window_size)
+  screen = pygame.display.set_mode ((0, 0), pygame.FULLSCREEN)
```
Comment (DannyPeck) [inline main.py]: Warning, pygame fullscreen mode is obnoxious to get out of.

**Comment by DannyPeck** [comment]
It appears this PR just prints the current selection, are we still waiting to implement the actually starting of the game?

File: main.py (hunk 1 above)
Comment (rupsis) [reply main.py]: the '\' key is the one that we've decided to dedicated as the "kill switch" key.

**Pull request merged**